Packs/CortexXDR/Playbooks/Cortex_XDR_Alerts_Handling_v2.yml

```diff
@@ -1512,21 +1536,91 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
-system: true
+  "19":
+    id: "19"
+    taskid: e322548d-8472-4958-8362-dd9aac91a0bf
+    type: playbook
+    task:
+      id: e322548d-8472-4958-8362-dd9aac91a0bf
+      version: -1
+      name: Cortex XDR - Large Upload
+      playbookName: Cortex XDR - Large Upload
+      type: playbook
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "5"
+    scriptarguments:
+      Alert_ID:
+        complex:
+          root: PaloAltoNetworksXDR.Incident.alerts
+          filters:
+          - - operator: containsGeneral
+              left:
+                value:
+                  simple: PaloAltoNetworksXDR.Incident.alerts.name
+                iscontext: true
+              right:
+                value:
+                  simple: Large Upload
+              ignorecase: true
+          accessor: alert_id
+          transformers:
+          - operator: uniq
+      AutoBlockIndicators:
+        simple: "True"
+      AutoIsolateEndpoint:
+        simple: "False"
+      BlockIndicators_UserVerification:
+        simple: "False"
+      EarlyContainment:
+        simple: "True"
+      FWApps_Processes_Whitlist:
+        simple: ip,tcp,udp,ssl,syslog,quic,Chrome.exe,Firefox.exe,Opera.exe,Safari.exe,iexplore.exe,msedge.exe,brave.exe
```

Comment:
Are the browsers not ignored within the alert itself? Assume this is an analytics alert...

Reply:
these safe browsers are not ignored by the XDR analytics